hack/vf-netns-switcher.sh

```diff
@@ -0,0 +1,343 @@
+#!/bin/bash
+
+conf_file=""
+
+declare -a netnses
+
+declare -A pfs
+declare -A pcis
+declare -A pf_port_names
+declare -A pf_switch_ids
+
+TIMEOUT="${TIMEOUT:-2}"
+POLL_INTERVAL="1"
```

Comment:
nit: make this configurable like timeout ?

Reply:
done